src/nupic/bindings/sparse_matrix.i

```diff
@@ -698,8 +709,8 @@ def __div__(self, other):
       // Return one list of triples
       toReturn = PyTuple_New(nnz);
       for (nupic::UInt ## N1 i = 0; i != nnz; ++i) {
-	PyObject* tuple = nupic::createTriplet ## N1(rows.get(i), cols.get(i), vals.get(i));
-	PyTuple_SET_ITEM(toReturn, i, tuple);
+  PyObject* tuple = nupic::createTriplet ## N1(rows.get(i), cols.get(i), vals.get(i));
```

Comment:
Indentation is wrong after conversion from tabs to spaces. Looks like you may have done find/replace so check for other places this might have happened.